packages/fuels/tests/types_predicates.rs

```diff
@@ -480,3 +480,107 @@ async fn spend_predicate_coins_messages_generics() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn spend_predicate_coins_messages_bytes() -> Result<()> {
```

Comment:
Could we make a helper for these tests and just alternate the data we pass to the predicate?

Reply:
Nice. Shortened the tests considerably. Resolving.